src/decks/study/StudyDeck.js

```diff
@@ -1,97 +0,0 @@
-import React, { useState, useEffect } from "react";
-import { useParams, useHistory } from "react-router-dom";
-import { readDeck } from "../../utils/api";
-import BreadCrumb from "../../common/BreadCrumb";
-
-function StudyDeck() {
-  const { deckId } = useParams();
-  const [flip, setFlip] = useState(true);
-  const [deck, setDeck] = useState({});
-  const [cardCount, setCardCount] = useState(0);
-  const [index, setIndex] = useState(0);
-
-  const history = useHistory();
-
-  useEffect(() => {
-    const abortController = new AbortController();
-
-    async function showCard() {
-      const cardList = await readDeck(deckId, abortController.signal);
-      setDeck(cardList);
-      setCardCount(cardList.cards.length);
-    }
-
-    showCard();
-    return () => abortController.abort();
```

Comment:
Not sure I understand the need of the AbortController here - could you expand on the use of this?